test/gtest/uct/test_pending.cc

```diff
@@ -137,4 +145,75 @@ UCS_TEST_P(test_uct_pending, pending_op)
     ASSERT_EQ(counter, iters);
 }
 
+UCS_TEST_P(test_uct_pending, send_ooo_with_pending)
+{
+    uint64_t send_data = 0xdeadbeef;
+    uint64_t test_pending_hdr = 0xabcd;
+    ucs_status_t status_send, status_pend = UCS_ERR_LAST;
+    ucs_time_t loop_end_limit;
+    unsigned i, counter = 0;
+
+    // TODO enable this test for ud once ooo in it is fixed.
+    if (GetParam()->tl_name == "ud" || GetParam()->tl_name == "ud_mlx5") {
+        UCS_TEST_SKIP;
+    }
+
+    initialize();
+    check_caps(UCT_IFACE_FLAG_AM_SHORT | UCT_IFACE_FLAG_PENDING);
+
+    /* set a callback for the uct to invoke when receiving the data */
+    uct_iface_set_am_handler(m_e2->iface(), 0, pending_am_handler , &counter);
+
+    loop_end_limit = ucs_get_time() + (ucs_time_from_sec(2) * ucs::test_time_multiplier());
```

Comment:
lets remove ucs::test_time_multiplier() - it can be too long with valgrind. better just let test pass.